Flake8 complains 
 ./resources/lib/channels/fr/tf1plus.py:438:103: W291 trailing whitespace
./resources/lib/resolver_proxy.py:269:39: E261 at least two spaces before inline comment

--- a/resources/lib/channels/fr/tf1plus.py
+++ b/resources/lib/channels/fr/tf1plus.py
@@ -435,5 +435,5 @@ def get_live_url(plugin, item_id, **kwargs):
 
     return resolver_proxy.get_stream_with_quality(plugin, video_url=video_url,
                                                   manifest_type="mpd", license_url=license_url,
-                                                  workaround=workaround, headers=headers_video_stream, 
+                                                  workaround=workaround, headers=headers_video_stream,
                                                   custom_license_headers=license_headers)

--- a/resources/lib/resolver_proxy.py
+++ b/resources/lib/resolver_proxy.py
@@ -266,7 +266,7 @@ def get_stream_with_quality(plugin,
     if license_url is not None:
 
         if '|' not in license_url:  # add headers only if they are not already in the url
-            if custom_license_headers: # use custom_license_headers only if it is set, else fallback to stream_headers
+            if custom_license_headers:  # use custom_license_headers only if it is set, else fallback to stream_headers
                 license_headers = urlencode(custom_license_headers)
             else:
                 license_headers = stream_headers

